src/products/mozilla_org/testing.md

```diff
@@ -123,36 +121,39 @@ This section focuses on instructions for testing pages with dynamically generate
 
 * Click the **Sign In to Monitor** button, you will be taken to the sign-in page in order to get to the [Firefox Monitor](https://monitor.firefox.com) site.
 * Click on the links under each of the following products:
-   * Firefox browser
-   * Firefox Lockwise
+   * Firefox Browser
    * Firefox Monitor
+   * Firefox Relay	
+   * Mozilla VPN
+   * Pocket
 
 Note: You may see different languages between mozilla.org, the login window, and the products above. If any of these products is not localized in your locale, it will fall back to other languages set as preferred for content language negotiation, assuming any of them is available for the project, or English.
 
 ### [firefox/all.ftl](https://www.mozilla.org/firefox/all/)
 
-On the lower left side of the page:
-* These two sites are in English only: [Check the system requirements](https://www.mozilla.org/firefox/system-requirements/) and [Release notes](https://www.mozilla.org/firefox/notes/).
-* Click on the **Source code** link, you will be directed to the topic on [MDN](https://developer.mozilla.org/docs/Mozilla/Developer_guide/Source_Code) site in your language if it is localized.
+On the right side of the page, there is a four-step process to go through in order to download a Firefox browser. 
+* Browsers: choose one of the options either for desktop or mobile. Click on the `?` next to the Browsers category, a popup window shows the descriptions of different versions of Firefox browsers available for download. Select one browser to proceed to the next step.
+* Platforms: Click on the `?`, a popup window shows the descriptions of different versions of installers available for download.
+* Languages: A list of currently available in the build depending on what is selected in the above two categories. Check whether your current preferred language is shown by default. Change to a different language, check whether the download summary above the **Download Now** button corresponds to your new selection.
+* Download: The **Download Now** button appears if the three categories are all available. 
```

Comment:
available?

Reply:
I was thinking of some smaller African languages for instance, depending on what one selected, especially in the platform, maybe that version is not available.

Reply:
Removed that conditional clause from the sentence.